_episodes/03-inventory.md

```diff
@@ -78,7 +78,7 @@ The mandatory columns and their formats are:
 - `checked out to`: project code - based on the list of `Users` in Snipe. can be blank if the asset is not yet assigned to a project.
 - `date shipped`: in format YYYY-MM-DD. can be blank
 - `expected return`: in format YYYY-MM-DD. can be blank
-- `notes`: freeform text. Please include transmitter ID (ex: A69-9001-12345) here if relevant
+- `notes`: freeform text. Please include transmitter ID (ex: A69-9001-12345) here if relevant (VR2ARs, VR2TXs, HR2s etc)
```

Comment:
I struggle with the grammar around "if relevant"... should there be a comma before it? (at the very least - we are missing consistency around this)

Reply:
how about "Please include transmitter ID (ex: A69-9001-12345) here, when relevant"